### ISSUE TITLE
Including Opdenjdk tz files

### ISSUE THREAD

**Issue by casper5822** [akashche/tzdbgen#2]
Hello, 
In the readme, there is the note about the openjdk files to include.
My question is how can i add the files with the tbgen?

Thank you

**Comment by akashche** [comment]
Hi, sorry, I don't remember all the details, I think it should read these [additional files](http://hg.openjdk.java.net/jdk8u/jdk8u/jdk/file/d0cccfd32b1e/make/data/tzdata), if they are added to `-srcdir` directory.

**Comment by casper5822** [comment]
Thank you, just other few questions:
1) Is this project alive? Do you still maintain it?
2) Does it work also for openjdk 12 ?
3) I don't understand how to pass this files on the srcdir

Thank you.

**Comment by akashche** [comment]
>    Is this project alive? Do you still maintain it?

No, the project is not maintained, its goal was to compile tzdata into jdk8+ format using jdk6 (that doesn't have NIO.2 and named regex used by tz compiler). With jdk6 not maintained anymore this project is likely not needed. To add customization to tz compiler - it is better to get its code straight from jdk8 (or jdk11 etc).

>    Does it also for openjdk 12 ?

AFAIK resulting tzdb format is stable and will work with any jdk or with standalone Joda time.

>    I don't understand how to pass this files on the srcdir

I think it reads all input files from the directory specified by `-srcdir` flag, but I don't know what exact logic is used to process these additional jdk-specific input files (`jdk11_backward` and some other).
